### PR TITLE
[TIMOB-19733] iOS9: Support "Continue" return key + new keyboards

### PR DESCRIPTION
**Pull request by hansemannn**
JIRA: https://jira.appcelerator.org/browse/TIMOB-19733

### CODE REVIEW

File: apidoc/Titanium/UI/UI.yml [hunk 1]
```diff
@@ -1507,9 +1507,31 @@ properties:
         `KEYBOARD_DEFAULT`.
     type: Number
     permission: read-only
-    
+
+  - name: KEYBOARD_WEBSEARCH
+    summary: Use a keyboard optimized for web search terms and URL entry.
+    description: |
+        Use with the <Titanium.UI.TextField.keyboardType> and <Titanium.UI.TextArea.keyboardType>
+        properties. This type features the space and “.” characters prominently.
```
Comment (cheekiatng) [inline apidoc/Titanium/UI/UI.yml]: running node validate in apidocs found error in this line.
Don't use "." here. use `.` instead.

**Comment by cheekiatng** [comment]
CR and FT passed. addressed documentation error and will merge.

**Comment by hansemannn** [comment]
Addressed comment

**Comment by cheekiatng** [comment]
APPROVED!

**Pull request merged**